.env.sample

```diff
@@ -1,4 +1,6 @@
 FAUNADB_SECRET=
 NEXT_PUBLIC_NODE_ADDRESS=https://cosmoshub.validator.network:443
 NEXT_PUBLIC_DENOM=uatom
-NEXT_PUBLIC_CHAIN_ID=cosmoshub-4
\ No newline at end of file
+NEXT_PUBLIC_DISPLAY_DENOM=ATOM
+NEXT_PUBLIC_DISPLAY_DENOM_EXPONENT=6
```

Comment:
I think you will need to add support for this value in the `printableCoins` function in `lib/displayHelpers.js`, so whenever we show a token amount it's rendered correctly.

Reply:
Yeah, good question how to best deal with this. Right now, `printableCoins` magically converts everything starting with `u`, which is good enough for all chains I am aware of. We can remove this automatic conversion and only use this configuration here to convert. But we can also use all 3: the configuration, the automation for `u` and the fallback to the original denom.

Reply:
In the last commit I extended `printableCoin` to also support the configuration and do not rely on auto-conversion only.